phaseshifts/backends.py

```
@@ -29,59 +29,155 @@ def autogen_from_input(self, bulk_file, slab_file, tmp_dir=None, **kwargs):
+    return viper_params, viper_phaseshifts, viper_poscar, runPhaseshiftGen
+
+
+def _load_native_eeasisss():
+    try:
+        from phaseshifts.lib.EEASiSSS import EEASiSSS as native_eeasisss
+    except ImportError:
+        return None
+    return native_eeasisss
+
+
+def _run_viperleed(parameters_file, slab_file, workdir, output_file):
```

Ruff (0.14.10)

Missing return type annotation for private function `_run_viperleed`

(ANN202)